hack/dockerfiles/generated-files.Dockerfile

```diff
@@ -5,6 +5,8 @@ FROM golang:1.16-buster AS gobuild-base
 # https://github.com/golang/protobuf/blob/v1.3.5/.travis.yml#L15
 ARG PROTOC_VERSION=3.11.4
 ARG GOGO_VERSION=v1.3.2
+ARG PROTOBUF_VERSION=v1.3.5
```

Comment:
nit, but arg should be right before the command that use them. So instead move the protoc/gogo arg to the correct place.

Reply:
Ah, yes, I can move these to where they're used (there's pros/cons for each; having them at the top allows to view all versions at a glance, so I though that was the intent). I saw the majority was defined at the top, so thought to move it there.

Let me update.